packages/synapse-react-client/src/lib/containers/personal_access_token/CreateAccessTokenModal.tsx

```diff
@@ -49,10 +44,7 @@ export const CreateAccessTokenModal: React.FunctionComponent<
     return !!tokenName && access.some(x => x)
   }
 
-  const onSubmit = async (
-    clickEvent: React.MouseEvent<HTMLElement, MouseEvent>,
-  ): Promise<void> => {
-    clickEvent.preventDefault()
+  const onSubmit = async (): Promise<void> => {
```

Comment:
`clickEvent.preventDefault` was throwing an error (`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'preventDefault')`) after refactoring to use `ConfirmationDialog` and updating the `onConfirm` type to allow passing an event.  I didn't see a noticeable change in the modal's performance after removing it, but wanted to double check if this is necessary here?

Reply:
Got it, thanks for helping me understand the context around using or not using preventDefault!